VideoAPI/VideoApi.DAL/Commands/RemoveMessagesForConferenceCommand.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using VideoApi.DAL.Commands.Core;
+using VideoApi.DAL.Exceptions;
+using VideoApi.Domain;
+using Task = System.Threading.Tasks.Task;
+
+namespace VideoApi.DAL.Commands
+{
+    public class RemoveMessagesForConferenceCommand : ICommand
```

Comment:
Another rename here. I'll have the ticket updated

Reply:
Renamed